From c17679b850857c82aa75278d2864ca512f9479b6 Mon Sep 17 00:00:00 2001
From: arathinx <ayyappanx.rathinam@intel.com>
Date: Fri, 27 Sep 2019 18:10:38 +0800
Subject: [PATCH] PONRTSYS-5192: GCC 8.3.0 warnings in cqm

Always add explicit {}s; even when they're not necessary too to avoid possible bugs. Resolved one checkpatch warnings
---
 drivers/net/ethernet/lantiq/cqm/prx300/cqm.c | 11 ++++++-----
 1 file changed, 6 insertions(+), 5 deletions(-)

diff --git a/drivers/net/ethernet/lantiq/cqm/prx300/cqm.c b/drivers/net/ethernet/lantiq/cqm/prx300/cqm.c
index a4474579746c..b15504841e6c 100644
--- a/drivers/net/ethernet/lantiq/cqm/prx300/cqm.c
+++ b/drivers/net/ethernet/lantiq/cqm/prx300/cqm.c
@@ -254,10 +254,11 @@ static u32 get_matching_pmac(u32 *ep, u32 flags, u32 *cbm_port, int pmac,
 						epg_lookup_table[i].pmac;
 						cbm_port[j] =
 						epg_lookup_table[i].epg;
-					if (cbm_port[j] ==
-							DMA_PORT_FOR_FLUSH)
-						dev_err(cqm_ctrl->dev, "DMA port %d in use pmac %d\n",
-							cbm_port[j], pmac);
+						if (cbm_port[j] ==
+							DMA_PORT_FOR_FLUSH) {
+							dev_err(cqm_ctrl->dev, "DMA port %d in use pmac %d\n",
+								cbm_port[j], pmac);
+						}
 						j++;
 					}
 				}
@@ -2684,7 +2685,7 @@ s32 qos_q_flush(s32 cqm_inst, s32 cqm_drop_port, s32 qid)
 					    CBM_PORT_F_DISABLE, 0);
 	cbm_w32((cqm_ctrl->deq + DQ_DMA_PORT(DMA_PORT_FOR_FLUSH, dqpc)), 0);
 	set_val((cqm_ctrl->deq + DQ_DMA_PORT(DMA_PORT_FOR_FLUSH, dptr)), 0,
-                DPTR_DMA_EGP_25_DPTR_MASK, DPTR_DMA_EGP_25_DPTR_POS);
+		DPTR_DMA_EGP_25_DPTR_MASK, DPTR_DMA_EGP_25_DPTR_POS);
 	return CBM_SUCCESS;
 }
 
